app/views/templates/search/search_object.rb

```diff
@@ -325,6 +326,16 @@ def labels
 
       private
 
+      def edm_is_shown_by_download_url
```

Comment:
Assignment Branch Condition size for edm_is_shown_by_download_url is too high. [15.43/15]